src/move/econia/sources/market.move

```diff
@@ -11994,7 +15554,7 @@ module econia::market {
         let min_base            = 0;
         let max_base            = 0;
         let min_quote           = 0;
-        let max_quote           = quote_trade + TICK_SIZE_COIN;
+        let max_quote           = quote_trade * TICK_SIZE_COIN;
```

Comment:
I kind of wish, instead of declaring all of these at the top of the test every time and copy pasting the block, that this were instead a structure returned by a default value method (or several, such as for bid/ask). Return the structure, and then any modifications made are clear and explicit. So basically you have known defaults and explicit deviations from those defaults. Right now, it's difficult to _understand_ this section in each test. What's different and interesting? Default struct + explicit mutations = much nicer.

Reply:
Good point on the default vs explicit mutations

I was avoiding abstractions to try and keep the test code simple to follow, e.g. so that we don't have to test the test code

But agreed that this style introduces some capacity for replicated errors

On this PR I don't think it's a worthwhile exercise to go back and abstract things

But on future testing suites there is certainly room for improvement